mitodl/open-discussions
# fix program list favorite

#### Pre-Flight checklist


#### What are the relevant tickets?

closes #2164 

#### What's this PR do?

this fixes an issue with unfavoriting the user lists in the favorites carousel on the course home page. see the issue for a longer description of the issues and repro steps.

#### How should this be manually tested?

checkout master and follow the repro steps on the issue to ensure you can repro the issue. then check this out and confirm it fixes the issue.
